src/modules/users/users.controller.ts

```diff
@@ -31,19 +31,90 @@ export class UsersController {
   @ApiOperation({ summary: 'Retrieve all admins' })
   @ApiResponse({ status: 200, description: 'List of all admins', type: [User] })
   @HttpCode(HttpStatus.OK)
-  findAll(
+  findAllAdmins(
```

Comment:
I dont see guard protection on controller or route.

Reply:
I created global guards, and they are in the App module.